tests/adapters/test_visa.py

```diff
@@ -42,21 +42,6 @@ def adapter():
                        read_termination="\n")
 
 
-def test_visa_version():
-    assert VISAAdapter.has_supported_version()
```

Comment:
Please also remove the method, and add a note to the changelog "Internal: Removed `VISAAdapter.has_supported_version()` as it is not needed anymore." -- in principle this was public-facing API , although I can't imagine anyone having used that.

Reply:
I already removed it.